account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp BV (http://therp.nl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    """Extend account.analytic.account."""
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def create_project_for_account(self):
+        """Create a project for an already existing analytic account."""
+        project_model = self.env['project.project']
+        for this in self:
+            if not this.projects_ids and this.type != 'view':
```

Comment:
@NL66278 I don't understand/ at least at this line
https://github.com/NL66278/account-analytic/blob/8.0-account_analytic_project/account_analytic_project/model/account_analytic_account.py#L16
it's `projects_ids`